lib/fibonacci.rb

```diff
@@ -5,8 +5,23 @@
 # ....
 # e.g. 6th fibonacci number is 8
 
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), n is the number of computations this calculation needs to run to compute the nth fibonacci number
```

Comment:
"the number of computations this calculation needs to run" is sort of what you're trying to define by explaining the "time complexity" of a method, so you don't want to use that in the definition.  "n" as you're using it here is actually the value of the input variable `n`. The time complexity is `n` because you're iteratively calculating each fibonacci number up to `n`.